app/javascript/components/util/EligibilityTooltip.js

```diff
@@ -1,136 +1,62 @@
 import React from 'react';
 import { PropTypes } from 'prop-types';
+import _ from 'lodash';
 
 import { formatTimestamp } from '../../utils/DateTime';
 import ReactTooltip from 'react-tooltip';
 
+const eligibility_options = [
+  { conditional: 'household', icon: 'fa-house-user', title: 'In a household:', messageCount: 1 },
```

Comment:
This component had a lot of repetitive code, so i converted it to an array of options/configs.